iron-collapse.html

```diff
@@ -127,7 +127,15 @@
       noAnimation: {
         type: Boolean
       },
-
+	  
+	  /**
+       * stores the desired size of the collapse body
+       *
+	   */
+	  _desiredSize: {
+		type: String,
+		value: ""
```

Comment:
seems there are some problems with the indentation in the file, also could you use single quotes in here?